index.html

```diff
@@ -4,9 +4,139 @@
+    </div>
+    
+    <div class="time secondary-font">2015.10.15(THU) ~ 16(FRI)</div>
+    <P class="venue primary-font">@ National Museum of Nigeria, Art Center Lagos and more</P>
+  </header>
+  <section class="main-program-section">
+    <div class="overlay">
+    <h1 class="program-head primary-font">
+      Main Program
+    </h1>
+    <div class="program-section">
+     
+      <div class="card-container">
+        <img src="./assets/images/Lecture.svg" alt="lucture logo" class="card-img">
+        <h5 class=" card-title primary-font">Lecture</h5>
+        <p class="card-text primary-font">Listen to the speakers from various countries about the messages of sharing and opening new ideas.</p>
+      </div>
+     <div class="card-container">
+        <img src="./assets/images/Exhibition.svg" alt="Exhibition logo" class="card-img">
+        <h5 class="card-title primary-font">CC Exhibition</h5>
+        <p class="card-text primary-font">Appreciate various creations applying CC license of artists, organized from Art Center Lagos</p>
+      </div>
+       <div class="card-container">
+        <img src="./assets/images/Forum.svg" alt="forum logo" class="card-img">
+        <h5 class="card-title primary-font">Forum</h5>
+        <p class="card-text primary-font">Have the time to share your thoughts and opinions for each topic.</p>
+      </div>
+      <div class="card-container">
+        <img src="./assets/images/Workshop.svg" alt="workshop logo" class="card-img">
+        <h5 class="card-title primary-font">Workshop</h5>
+        <p class="card-text primary-font">Try creating your own work using open source license rather than just watching at it.</p>
+      </div>
+      <div class="card-container">
+        <img src="./assets/images/ignite.svg" alt="ignite logo" class="card-img">
+        <h5 class="card-title primary-font">CC Ignite</h5>
+        <p class="card-text primary-font">get opportunities to network with CC affiliates around the world, also after the summit.</p>
+      </div>
+    </div>
+    <a href="#" class="link primary-font">Join CC Global Submit 2015</a>
+    <a href="" class=" fullProgram tetiary-font">SEE THE WHOLE PROGRAM</a>
+
+  </div>
+  </section>
+  <h3 class="featured primary-font"> Featured speakers</h3>
+  <section class="speaker-section" >
+    <div class="speaker-details " id="speaker-details">
+      
+    </div>  
+    
+  </section>
+  <div class="down-btn">
+
+    <button type="button" class="more-future primary-font" id="showMore"><p class="more-btn">More</p>  
+    </button> 
+  </div>
+  <footer id="partners">
+    <div class="home-partner-section">
+      <h1 class="footer-title tetiary-font">Partner</h1>
+      <div class="partner-link">
+        <a href="" class="mozilla"></a>
+        <a href="" class="google"></a>
+        <a href="" class="naver"></a>
```

Comment:
- [ ] Great job adding the `partners' section` and the` footer section` as the previous reviewer mentioned. I will suggest that in the partner's section, you reduce the spacing between the logos of the partners so your design looks more similar to the original design :100: 
- [ ] Also there is a small orange horizontal line under the section title `Partners` which you are missing in your design so kindly include that as well :+1: 
- [ ] Kindly ensure that there is a similar spacing between the `speakers section` and the `partners section` as seen in the original design. Currently you have a very small space between them both :+1: 
- [ ] Lastly, Please ensure  that the color of the section title `Partners` is the same as that of the original Design :100: 

| Your Design | The Original Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/227723321-17456367-4bc2-429d-927e-b4ea84adf5f3.png) | ![image](https://user-images.githubusercontent.com/102699958/227723280-23c9b259-3d00-4152-b6ac-883e272bcc09.png) |
| ![Screenshot from 2023-03-25 14-29-38](https://user-images.githubusercontent.com/102699958/227723534-c81f965d-7bf5-494a-a140-6219389158ae.png) | ![Screenshot from 2023-03-25 14-31-26](https://user-images.githubusercontent.com/102699958/227723608-eea59139-67c3-439e-a2fc-367100a12d16.png) |